api/cloudcontroller/ccv3/relationship_list_test.go

```diff
@@ -92,4 +92,95 @@ var _ = Describe("RelationshipList", func() {
 			})
 		})
 	})
+
+	Describe("ShareServiceInstanceToSpaces", func() {
+		var (
+			serviceInstanceGUID string
+			spaceGUIDs          []string
+
+			relationshipList RelationshipList
+			warnings         Warnings
+			executeErr       error
+		)
+
+		BeforeEach(func() {
+			serviceInstanceGUID = "some-service-instance-guid"
+			spaceGUIDs = []string{"some-space-guid", "some-other-space-guid"}
+		})
+
+		JustBeforeEach(func() {
+			relationshipList, warnings, executeErr = client.ShareServiceInstanceToSpaces(serviceInstanceGUID, spaceGUIDs)
+		})
+
+		Context("when the delete is successful", func() {
```

Comment:
please change the spec to: when no errors are encountered

Reply:
👍